README.md

```diff
@@ -173,6 +173,12 @@ RUST_BACKTRACE=1 RUST_LOG=debug gooddata-fs joe.doe@gooddata.com secretpassword
 
 ### Docker
 
+Make the parent mount point 'shared'. For example, if the gooddata mountpoint
+is `/mnt/gd-fs`, do:
+```
+sudo mount --make-shared $(df --output=target /mnt/gd-fs | tail -n1)
+```
+
```

Comment:
Bad place and out of context.. I'll reorder the docker-related docs in another commit.